pyshtools/shclasses/shcoeffsgrid.py

```diff
@@ -717,6 +806,40 @@ def to_file(self, filename, format='shtools', header=None, **kwargs):
             raise NotImplementedError(
                 'format={:s} not implemented.'.format(repr(format)))
 
+    def to_netcdf(self, filename, title='', description=''):
```

Comment:
I think that it would be a good idea to be able to specify an optional `lmax` to read, like the `from_netcdf()` method above.

Reply:
Generally I agree, but I think this should be done consistently for all to_xxx() functions then. I'll leave it like this if that's fine.